src/mame/drivers/jpmsru.cpp

```diff
@@ -353,12 +445,22 @@ void jpmsru_state::out_payout_token_w(offs_t offset, uint8_t data)
 
 void jpmsru_state::out_coin_lockout_w(offs_t offset, uint8_t data)
 {
+	m_coin_lockout = !data;
 	machine().bookkeeping().coin_lockout_w(0, !data);
 	machine().bookkeeping().coin_lockout_w(1, !data);
 	machine().bookkeeping().coin_lockout_w(2, !data);
 	machine().bookkeeping().coin_lockout_w(3, !data);
 }
 
+void jpmsru_state::out_50p_lockout_w(offs_t offset, uint8_t data)
+{
+	// 50p is always coin 4
+	if(!m_coin_lockout)
+		machine().bookkeeping().coin_lockout_w(3, !data);
+	else
+		machine().bookkeeping().coin_lockout_w(3, 1);
+}
```

Comment:
This seems weird – is this output really interlocked so it can only be driven low when the other coin lockout port is all high?

Reply:
Seemed like it, but now that I've tested things a bit more, looks like I was fixing a BTANB here. On j_ews you can go over the
credit limit with 50p coins without the check, that's the only game where it happens though and it got fixed on later
revisions. No idea how this looks like in hardware, no schematics for those parts. Removed the check.